tests/test_loading.py

```diff
@@ -268,6 +270,13 @@ def test_load_gt2numpy_by_pixels(self):
         data = dc.load_by_pixels(self.row, self.col, row_size=self.row_size, col_size=self.col_size, dtype='numpy')
         assert np.array_equal(self.ref_np_ar_area, data)
 
+    def test_load_by_pixels_doc_example(self):
+        dc = self._create_loadable_dc(self.gt_filepaths)
+        # begin-snippet: data_cube_load_numpy_by_pixels
+        data = dc.load_by_pixels(970, 246, row_size=10, col_size=16, dtype='numpy')
+        # end-snippet
+        assert np.array_equal(self.ref_np_ar_area, data)
```

Comment:
But here we have the same issue again, because `self.ref_np_ar_area ` changes when changing the class' row and column variables. So either we simply assign `True` so that we can check if the test runs at all. Or we create a second, decoupled `self.ref_np_ar_area ` variable here.

Reply:
I thought about it, but I'd only change this once it becomes necessary, to avoid code duplication. When ref_no_ar_area changes this test will fail and it is the author of the changes responsibility to think about a good solution - at that time one has also the advantage of knowing the context what caused the ref_np_ar_area change, which might also warrant a change in documentation and examples. IMO the decoupling is already achieved by providing two separate tests where one can make changes once this becomes necessary.